plugins/obs-ffmpeg/CMakeLists.txt

```diff
@@ -18,8 +18,25 @@ add_library(OBS::ffmpeg ALIAS obs-ffmpeg)
 
 add_subdirectory(ffmpeg-mux)
 if(ENABLE_NEW_MPEGTS_OUTPUT)
-  find_package(Librist REQUIRED)
-  find_package(Libsrt REQUIRED)
+  find_package(Librist QUIET)
+  find_package(Libsrt QUIET)
+
+  if(NOT TARGET Librist::Librist AND NOT TARGET Libsrt::Libsrt)
+    obs_status(
+      FATAL_ERROR
+      "Srt and librist libraries not found! Please install srt and librist or set ENABLE_NEW_MPEGTS_OUTPUT=OFF."
+    )
+  elseif(NOT TARGET Libsrt::Libsrt)
+    obs_status(
+      FATAL_ERROR
+      "Srt library not found! Please install srt or set ENABLE_NEW_MPEGTS_OUTPUT=OFF."
+    )
+  elseif(NOT TARGET Librist::Librist)
+    obs_status(
+      FATAL_ERROR
+      "Librist library not found! Please install librist or set ENABLE_NEW_MPEGTS_OUTPUT=OFF."
```

Comment:
Use SRT or libsrt and RIST or librist.

Reply:
`srt` seems to be commonly used as package name. So I guess `srt/libsrt` would may be the best solution.

Reply:
SRT and RIST are acronyms and should be capitalized when used as such. The terms libsrt and libelist are not acronyms, so they do not require special capitalization. Furthermore, "SRT" and "RIST" indicating that they are acronyms helps provide a hint of what your desired search results should be, whereas "srt" and "rist" are not as clear. Please capitalize them the way that I have requested.

Reply:
Should be good now.